_overviews/contributors/index.md

```diff
@@ -0,0 +1,809 @@
+---
+layout: singlepage-overview
+title: Library Author Guide
+discourse: true
+---
+
+This document targets developers who want to publish their work as a library that other programs can depend on.
+The document steps through the main questions that should be answered before publishing an open source library,
+and shows how a typical development environment looks like.
+
+An example of library that follows the recommendations given here is available at
+[https://github.com/scalacenter/library-example](https://github.com/scalacenter/library-example). For the sake
+of conciseness, this example uses mainstream technologies like GitHub, Travis CI, and sbt, but alternative
+technologies will be mentioned and adapting the contents of this document for them should be straightforward.
+
+## Choose an Open Source License
+
+The first step consists in choosing an open source license specifying under which conditions the library
+can be reused by other people. You can browse the already existing open source licenses on the
+[opensource.org](https://opensource.org/licenses) website. If you don’t know which one to pick, we suggest
+using the [Apache License 2.0](https://opensource.org/licenses/Apache-2.0), which allows users to use (including
+commercial use), share, modify and redistribute (including under different terms) your work under the condition
+that the license and copyright notices are preserved.
+
+Once you have chosen a license, *apply* it to your project by creating a `LICENSE` file in the root directory
+of your project with the license contents or a link to it. This file usually indicates who owns the copyright.
+We suggest putting the project “contributors” as copyright owners. You can see our
+[LICENSE file](https://github.com/scalacenter/library-example/blob/master/LICENSE) as an example.
+
+## Host the Source Code
+
+We recommend to share the source code of your library by hosting it on a public [Git](https://git-scm.com/)
```

Comment:
nbd, but "recommend to X" sounds off to my native-speaker ears. "recommend sharing" or "recommend that you share" would be fine